packages/varlet-ui/src/popup/__tests__/index.spec.js

```diff
@@ -85,30 +85,83 @@ test('test popup close on clickOverlay', async () => {
   wrapper.unmount()
 })
 
-test('test popup default style', async () => {
+test('test popup default style', () => {
   const wrapper = mount(Wrapper, {
     props: {
-      defaultStyle: false,
+      show: true,
+      defaultStyle: true,
```

Comment:
The defaultStyle defaults true, We should test the case for other value.

Reply:
get

Reply:
> The defaultStyle defaults true, We should test the case for other value.

done